tac.md

```diff
@@ -0,0 +1,56 @@
+---
+---
+
+tac
+--
+
+'tac' concatenates each file or command line input to standard output in reverse (hence tac = cat in reverse).
+
+~~~ bash
+$ tac filename
+~~~
+
+If a filename is provided, 'tac' prints the contents of the file line-by-line, starting with the last line first.
+
+If no filename is provided, or the filename is inputted as "-", 'tac' reads input from the command line and prints it in reverse.
+
+### Additional Options/Examples
+
+~~~ bash
+$ tac [OPTION] [FILE]
+~~~
+
+By default, line separators are newlines and are placed after each line of output.  Use '-b' to place line separators before each line of output instead.
```

Comment:
Please surround all command line arguments or excerpts in \`backticks\` so they appear `like this`.